app/src/main/res/values-hr-rHR/strings.xml

```diff
@@ -1,114 +1,114 @@
     <!-- Error messages -->
-    <string name="chown_fail">Failed to `chown` APK to system owner, please try again.</string>
-    <string name="error_downloading">Error Downloading %1$s</string>
-    <string name="failed_uninstall">Failed to uninstall package %1$s</string>
-    <string name="files_missing_va">Failed to locate the required files for installation. Re-download the installation files, then try again.</string>
-    <string name="ifile_missing">Failed to locate apk file for black/dark theme from storage, please try again.</string>
-    <string name="installation_aborted">Installation failed because the user aborted the installation.</string>
-    <string name="installation_blocked">Installation failed because the user blocked the installation.</string>
-    <string name="installation_downgrade">Installation failed because the user tried to downgrade the package. Uninstall updates from the stock YouTube app, then try again.</string>
-    <string name="installation_conflict">Installation failed because of the app conflicts with an already installed app. Uninstall the current version of Vanced, then try again.</string>
-    <string name="installation_failed">Installation failed for unknown reasons, join our Telegram or Discord for further support.</string>
-    <string name="installation_incompatible">Installation failed because the installation file is incompatible with your device. Clear downloaded files in the Settings, then try again.</string>
-    <string name="installation_invalid">Installation failed because the apk files are corrupted, please try again.</string>
-    <string name="installation_signature">Installation failed because apk signature verification is enabled. Disable apk signature verification, then try again.</string>
-    <string name="installation_miui">Installation failed because MIUI Optimization is enabled. Disable MIUI Optimization, then try again.</string>
-    <string name="installation_storage">Installation failed due to a storage error.</string>
-    <string name="modapk_missing">Failed to find apk file for black/dark theme from the installer. Clear app data of Manager, then try again.</string>
-    <string name="path_missing">Failed to locate the stock YouTube installation path after split installation.</string>
+    <string name="chown_fail">Neuspjela promjena `chown` APK u vlasnika sustava, pokušajte ponovno.</string>
+    <string name="error_downloading">Greška preuzimanja %1$s</string>
+    <string name="failed_uninstall">Neuspjela deinstalacija %1$s paketa</string>
+    <string name="files_missing_va">Neuspjelo lociranje potrebnih datoteka za instalaciju. Ponovno preuzmite datoteke za instalaciju, zatim ponovno pokušajte.</string>
+    <string name="ifile_missing">Neuspjelo lociranje apk datoteke za crnu/tamnu temu iz pohrane, pokušajte ponovno.</string>
+    <string name="installation_aborted">Instalacija je prekinuta zato jer ju je korisnik prekinuo.</string>
```

Comment:
Proper translation would be:
Instalacija je prekinuta zato što ju je korisnik prekinuo.
OR
Instalacija je prekinuta jer ju je korisnik prekinuo.

Both of these are fine, but using "zato jer" is pleonastic.